SPECS-EXTENDED/perl-XML-LibXSLT/perl-XML-LibXSLT.spec

```diff
@@ -0,0 +1,331 @@
+Summary:        Perl module for interfacing to GNOME's libxslt
+Name:           perl-XML-LibXSLT
+# NOTE: also update perl-XML-LibXML to a compatible version.  See below why.
```

Comment:
Is this comment still valid? I don't see the "See below why" part.:)

Reply:
Fixed ...